.vscode/settings.json

```diff
@@ -19,10 +19,10 @@
   },
   "files.exclude": {
     ".github": true,
-    "builder/build": true,
-    "components": true,
-    "drivers": true,
-    "esp-idf": true,
+    // "builder/build": true,
+    "components": false,
+    // "drivers": true,
+    // "esp-idf": true,
     "xtensa-esp32-elf": true
```

Comment:
```suggestion

```

(after deleting xtensa-esp32-elf)

Reply:
I haven't noticed a slowdown yet. It's nice to be able to click through all the component references. And those other folders don't exist anymore. I should delete those lines.